docs/user/advanced_features.rst

```diff
@@ -175,8 +175,32 @@ setting for ``simu.reloadH5Data`` in the WEC-Sim input file.
     Please use ``userDefinedFunctions.m`` instead.
 
 
+Radiation Force calculation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The radiation forces are calculated at each time-step by the convolution of the body velocity and the 
+radiation Impulse Response Function -- which is calculated using the cosine transform of the radiation-damping.
+Speed-gains to the simulation can be made by using alternatives to the convolution operation, namely, 
+
+1. The State-Space Representation,
+2. The FIR Filters.
+
+Here is a comparison of simulation run-times for the RM3 example simulated for 1000 s, when the radiation forces 
+are calculated using different routes:
+
+	+------------------------------------------------+-------------------------------------------+
+	|   *Radiation Force Calculation Approach*       | *Time (s)*  	                             |
+	+------------------------------------------------+-------------------------------------------+
+	|   No Convolution                               | 3.699 s    	                             |
```

Comment:
@salhus Minor comment: can we call No Convolution "Constant Coefficients" as this is what the block is called in the Wave Radiation Forces Calculation Block.

Second when I run the simulation I get slightly different times, so I wanted to request we report normalized times. So No Convolution would be 1 and all other times would be referenced to that. I think there will be confusion if these numbers are not reproduced by users unfortunately there is a lot of factors impacting the final values.

Reply:
Updated the docs.
- done